setup.cfg

```diff
@@ -70,6 +70,7 @@ install_requires =
     scikit-image>=0.19.1
     scipy>=1.4.1 ; python_version < '3.9'
     scipy>=1.5.4 ; python_version >= '3.9'
+    sphinx<5
```

Comment:
why do we need this in our runtime dependencies?

Reply:
Because sphinx is dependencies of numpydoc that is in napari dependencies. 

But there should be commentabout this next to this entry.

Reply:
Is numpydoc also runtime dependency?
I have a personal project that uses sphinx = "^6.1.3" and am having trouble adding Napari because of the version conflict. I know this is a specific case, but it would help for docs-related dependencies to be in extras.

Reply:
Please open a separate issue, I doubt comment on older-merged PRs will be seen/worked on.

Reply:
@AtomScott we are actually planning to make a `[docs]` install for all docs related dependencies (and those for the examples), so for sure please make an issue and we'll make sure it's addressed in the new PR.

Reply:
At any rate since I don't see the new issue (?), yes, numpydoc is a runtime dependency (run `git grep numpydoc` to check), because we use it to fiddle with docstrings in some of our autogenerated code. We may be able to remove this at some point.